app/controllers/answers_controller.rb

```diff
@@ -0,0 +1,34 @@
+class AnswersController < ApplicationController
+  # before_action :set_answer
+
+  def create
+    @answer = Answer.new(answer_params)
+    @answer.user = current_user
+
+    respond_to do |format|
+      if @answer.save
+        format.html { redirect_to question_path(@answer.question, {:id => @answer.question}), notice: 'Answer was successfully created.' }
+      else
+        format.html { redirect_to question_path(@answer.question), notice: 'Could not create answer.' }
+      end
+    end
+  end
+
+  def update
+  end
+
+  def destroy
+  end
+
+  private
+
+  # def set_answer
```

Comment:
If you want to use `#set_answer`, you can specify the `before_action` to individual routes like:
```ruby
before_action :set_answer, only: [:update, :destroy]
```